web/src/components/dashboard/SidebarCardsContainer.test.tsx

```diff
@@ -223,11 +226,12 @@ describe("<SidebarCardsContainer />", () => {
       expect(screen.queryByText("Cert 3")).not.toBeInTheDocument();
     });
 
-    it("links to task page for certifications", () => {
+    it("routes to task page for certifications", () => {
```

Comment:
seems like this test was copy-pasted / misnamed... it's not a "task" page and we should edit that.

Reply:
renamed